packages/storybook/src/web-components/button/button.mdx

```diff
@@ -6,7 +6,9 @@ import * as ButtonStories from "./button.stories";
 
 <cite className="lsb-citation">
   Documentatie afkomstig van
-  <a href="https://nl-design-system.github.io/utrecht/storybook-css/index.html?path=/docs/css-button--docs">Utrecht Button</a>
+  <a href="https://nl-design-system.github.io/utrecht/storybook-css/index.html?path=/docs/css-button--docs">
```

Comment:
Prettier maakt hier een zooitje van de output...: 
```
<cite class="lsb-citation"><p>Documentatie afkomstig van</p><a href="https://nl-design-system.github.io/utrecht/storybook-css/index.html?path=/docs/css-button--docs"><p>Utrecht Button</p></a></cite>
```
Misschien helpt het om er `{` en `}`, zie ook [JSX in MDX: A Guide to Removing Unwanted HTML Paragraph Tags](https://jasongerbes.com/blog/mdx-remove-unwanted-paragraph-tags#use-a-javascript-expression)

Reply:
Aangepast naar semantische HTML en een `/* prettier-ignore */`